charts/linstor-scheduler-extender/templates/NOTES.txt

```diff
@@ -0,0 +1,10 @@
+Scheduler {{ include "linstor-scheduler-extender.fullname" . }} deployed!
+
+Please run `helm test {{ .Release.Name }}` to ensure it's properly working.
+
+Specify the scheduler on your pods to start smart scheduling based on your Persistent Volumes:
+
+---
+spec:
+  schedulerName: {{ include "linstor-scheduler-extender.fullname" . }}
```

Comment:
I installed chart with the following options:

```
helm install linstor-scheduler ./linstor-scheduler-extender --namespace lisntor --set linstorEndpoint=https://piraeus-op-cs.linstor.svc:3371 --set linstorClientSecret=piraeus-op-client-secret
```

So resulting name of scheduler was
```
linstor-scheduler-linstor-scheduler-extender
```

I'd like to see something more usable value eg. `linstor` `piraeus` or `linstor-scheduler`, `piraeus-scheduler`

Reply:
I'd like to keep the "dependency" on the chart name, but if we just rename the whole chart `linstor-scheduler` that should do the trick. There is some helm magic regarding "release name == chart name" for this case.

Reply:
I'm OK if it will be `linstor-scheduler` (release-name) but not `linstor-scheduller-linstor-scheduller` (chart-name + release-name) ;)

Reply:
Helm logic seems to be:
* If chart name is contained in release name -> use just release name
* If chart name is not fully contained in release name -> use release-name+chart name

So if we just use `linstor-scheduler` everywhere, that should be good enough